### PR TITLE
Fixing bug with user control binding

### PR DESCRIPTION
**Pull request by SaviorXTanren**

### CODE REVIEW

File: MixItUp.WPF/Controls/Chat/ChatUserControl.xaml.cs [hunk 1]
```diff
@@ -14,31 +16,61 @@ public partial class ChatUserControl : UserControl
     {
         public UserViewModel User { get; private set; }
 
+        public ChatUserControl()
+        {
+            this.DataContextChanged += ChatUserControl_DataContextChanged;
```
Comment (TyrenDe) [inline MixItUp.WPF/Controls/Chat/ChatUserControl.xaml.cs]: I would put these after the InitalizeComponent();

File: MixItUp.WPF/Controls/Chat/ChatUserControl.xaml.cs [hunk 2]
```diff
@@ -14,31 +16,61 @@ public partial class ChatUserControl : UserControl
...
+
+            InitializeComponent();
+        }
+
         public ChatUserControl(UserViewModel user)
```
Comment (TyrenDe) [inline MixItUp.WPF/Controls/Chat/ChatUserControl.xaml.cs]: Not sure how you feel about "chaining" constructors together but you could do:
public Foo() {}
public Foo(Bar b) : this() {}

Comment (SaviorXTanren) [reply MixItUp.WPF/Controls/Chat/ChatUserControl.xaml.cs]: Yea, makes more sense now. Didn't make as much with the previous path.

File: MixItUp.WPF/Controls/Chat/ChatUserControl.xaml.cs (hunk 1 above)
Comment (SaviorXTanren) [reply MixItUp.WPF/Controls/Chat/ChatUserControl.xaml.cs]: Sounds good

**Pull request merged**